x-pack/test/security_solution_cypress/cypress/e2e/investigations/threat_intelligence/indicators.cy.ts

```diff
@@ -358,10 +358,11 @@ describe('Invalid Indicators', { tags: ['@ess'] }, () => {
   });
 });
 
-describe('Missing mappings', { tags: ['@ess'] }, () => {
-  before(() => cy.task('esArchiverLoad', { archiveName: 'ti_indicators_data_no_mappings' }));
+// TODO: investigate and re-enable this
```

Comment:
Are you intending to reenable this before merging or do this as part of a follow up PR?